From 842275f245e488342f5d8ad854a7a156c92711dc Mon Sep 17 00:00:00 2001
From: Nahuel Gomez Castro <nahual_gomca@outlook.com.ar>
Date: Thu, 9 Jul 2020 22:28:43 -0300
Subject: [PATCH] Fixed compilation with Vala 0.48.7

---
 json/Array.vala | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/json/Array.vala b/json/Array.vala
index 52408bc..0460a11 100644
--- a/json/Array.vala
+++ b/json/Array.vala
@@ -76,8 +76,10 @@ namespace GJson {
 			}
 			if (sc.items.type().is_a (typeof (JsonSchema.Schema))) {
 				var s = (JsonSchema.Schema)sc.items;
-				foreach (var item in this)
+				this.foreach (item => {
 					item.validate (s);
+					return true;
+				});
 			}
 			else if (sc.items.type().is_a (typeof (JsonSchema.SchemaList))) {
 				var sl = (JsonSchema.SchemaList)sc.items;
--
libgit2 1.0.0

