From 2167aa0849358e6ea0f60c2dc23cbba0f558ad8d Mon Sep 17 00:00:00 2001
From: Aleks Rutins <aleks@rutins.com>
Date: Sat, 29 Oct 2022 17:33:18 -0400
Subject: [PATCH] Fix #259

---
 src/documentation/girdocumentation.vala | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/documentation/girdocumentation.vala b/src/documentation/girdocumentation.vala
index 36e8b7d7..04b9f01d 100644
--- a/src/documentation/girdocumentation.vala
+++ b/src/documentation/girdocumentation.vala
@@ -280,7 +280,7 @@ class Vls.GirDocumentation {
      * see [[https://developer.gnome.org/gtk-doc-manual/stable/documenting_syntax.html.en]]
      */
     public string render_gtk_doc_content (string content, Vala.Comment comment, Compilation compilation) throws GLib.RegexError {
-        string comment_data = content;  // FIXME: workaround for valac codegen bug
+        string comment_data = content.dup();  // FIXME: workaround for valac codegen bug
 
         // replace code blocks
         comment_data = /\|\[(<!-- language="(\w+)" -->)?((.|\s)*?)\]\|/
